src/variants/beamos/config.beta

```diff
@@ -3,18 +3,21 @@
 # Will add the dev environment and pre-set the dev user.
 # export BEAMOS_DEV_CONFIG=yes
 
-export OCTOPI_OCTOPRINT_REPO_BRANCH=mrbeam2-beta
+export OCTOPI_OCTOPRINT_REPO_BRANCH=beta
 
-export BEAMOS_MRBEAMPLUGIN_REPO_BRANCH=mrbeam2-beta
+export BEAMOS_MRBEAMPLUGIN_REPO_BRANCH=beta
 export BEAMOS_CAMERA_REPO_BRANCH=mrbeam2-beta
-export BEAMOS_IOBEAM_REPO_BRANCH=mrbeam2-beta
-export BEAMOS_MRBEAMLED_REPO_BRANCH=mrbeam2-beta
+export BEAMOS_FINDMYMRBEAM_REPO_BRANCH=beta
+export BEAMOS_NETCONNECTD_PLUGIN_REPO_BRANCH=beta
```

Comment:
recheck if beta makes sense

Reply:
should be a beta tag, but as long as we don't have a automation we can use the branch for the plugins